src/content/platform-integration/ios/setup-ios.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Install and set up for iOS development
+description: Configure your system to develop Flutter for iOS.
+short-title: Set up iOS
+---
+
+## Install Flutter
+
+To set up your development environment for iOS,
+you first need to install Flutter on your
+development machine.
+
+If you have already installed Flutter,
+ensure that it's [up to date][].
+
+:::note Did you know?
+When you install or update the Flutter framework,
+it also installs and updates the Dart language for you.
+:::
+
+If Flutter is installed and up to date,
+continue to the next step.
+
+[up to date]: /install/upgrade
+
+## Install Xcode
+
+With Xcode, you can run Flutter apps on
+an iOS physical device or on the iOS Simulator.
+
+If you haven't done so already,
+install and set up [Xcode][].
+If you have installed Xcode, ensure that
+it's up to date and continue to the next step.
+(After setting up the command-line tools
+and accepting the license, as shown here.)
+
+[Xcode]: https://developer.apple.com/xcode/
+
+To configure Xcode's command-line tools,
+use the following command:
+
+```console
+$ sudo sh -c 'xcode-select -s /Applications/Xcode.app/Contents/Developer && xcodebuild -runFirstLaunch'
+```
+
+Sign the Xcode license agreement:
+
+```console
+$ sudo xcodebuild -license
+```
+
+## Configure your target iOS device
+
+Note that the behavior of the iOS Simulator can vary
+from an actual physical device. It's easier to get
+the Simulator up and running than a physical device,
+which requires more setup.
+For that reason, we recommend starting with the Simulator.
+
+{% tabs "ios-simulator-or-physical-device" %}
+{% tab "Simulator" %}
+
+### Configure your iOS Simulator
+
+Use the following steps:
+
+1. To install the iOS Simulator, run the following command.
+
+    ```console
+    $ xcodebuild -downloadPlatform iOS
+    ```
+
+1. Start the Simulator with the following command:
+
+    ```console
+    $ open -a Simulator
+    ```
+
+1. Set your Simulator to use a 64-bit device.
+   This covers iPhone 5s or later.
```

Comment:
I think you can drop this note. I believe we don't support any devices before the iPhone 5s.

```suggestion
```